src/test/kotlin/org/jitsi/jibri/api/http/HttpApiTest.kt

```diff
@@ -66,6 +66,8 @@ class HttpApiTest : ShouldSpec() {
             override fun configure(): Application {
                 return ResourceConfig(object : ResourceConfig() {
                     init {
+                        // Uncommenting the following line can help with debugging any errors
+                        //property(LoggingFeature.LOGGING_FEATURE_LOGGER_LEVEL_SERVER, "WARNING")
```

Comment:
Do you think it's useful to leave this kind of debugging/test-help in the test itself. Does this tip not apply to tests in general?

Reply:
i thought so because i had an error here and it took some digging to find the right function to enable the debug logging for jersey (which was necessary to figure out what was going wrong)

Reply:
And HttpApiTest is the only test that exercises jersey? And jersey cannot be abstracted away, I assume.

Reply:
right, HttpApiTest actually uses this test jersey framework that makes it really go through the requests (rather than just calling the HttpApi methods directly) which is really nice because a lot of stuff happens there.